scranShipper/src/Basket/BasketItem/index.jsx

```diff
@@ -0,0 +1,24 @@
+const BasketItem = (props) => {
+
+    const {updateOrder} = props
```

Comment:
If you're destructuring `updateOrder` here, might as well do it for all the props and do it in the parameters